pkg/builders/api_version_builder.go

```diff
@@ -29,6 +29,11 @@ type VersionedApiBuilder struct {
 	Kinds         []*versionedResourceBuilder
 	GroupVersion  schema.GroupVersion
 	SchemeBuilder runtime.SchemeBuilder
+
+	// Deprecated
+	// Only for compiliation backward-compatibility w/ 1.12+ version generators
+	// removing in the future
```

Comment:
One tiny nitpick is that `Deprecated:` is the marker described in https://github.com/golang/go/wiki/Deprecated

Reply:
thanks, updated